src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -136,22 +154,9 @@ class TableView extends React.Component<TableViewProps> {
           </StyledLink>
         </Tooltip>,
       ];
+    } else {
+      return [];
```

Comment:
Is this reachable? Shouldn't we always render the id field as it will always be in the result if there are no aggregates and we want to make sure users have a way to look at individual events.

Reply:
Yes, this happens when there aren't any aggregates and it has the id columns selected. When that happens, we choose not to prepend the additional id column in favour of the selected id column.

Reply:
A good follow up to this PR would be to give the field renderer for the id column the ability to go to the event itself like the prepended id column.

Reply:
That sounds like a great idea.

Reply:
Sounds good, I'll look into that soon 👍